apps/percona/workload/test.yml

```diff
@@ -0,0 +1,117 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+       ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - name: Generate the litmus result CR to reflect SOT (Start of Test) 
+          template: 
+            src: /litmus-result.j2
+            dest: litmus-result.yaml
+          vars: 
+            test: "{{ test_name }}"
+            app: ""
+            chaostype: ""
+            phase: in-progress
+            verdict: none
+        
+        - name: Apply the litmus result CR
+          shell: kubectl apply -f litmus-result.yaml
+          args:
+            executable: /bin/bash
+          register: lr_status 
+          failed_when: "lr_status.rc != 0"
+
+        - name: Getting the Sevice IP of Application
+          shell: kubectl get svc -n {{ namespace }} -l {{ app_service_label }} -o jsonpath='{.items[0].spec.clusterIP}'
+          register: ip  
+
+        - name: Replace the Service IP placeholder
+          replace:
+            path: "{{ percona_loadgen }}"
+            regexp: "service_ip"
+            replace: "{{ ip.stdout }}"
+
+        - name: Create test specific namespace.
+          shell: kubectl create ns {{ namespace }}
+          args:
+           executable: /bin/bash
+          when: namespace != 'litmus'
+
+        - name: Checking the status of test specific namespace.
+          shell: kubectl get ns {{ namespace }} -o jsonpath='{.status.phase}'
+          args:
+           executable: /bin/bash
+          register: npstatus
+          until: "'Active' in npstatus.stdout"
+          delay: 30
+          retries: 10
+
+        - name: Checking for configmap  
+          shell: kubectl get configmap -n {{ namespace }}
+          register: configmap
+
+        - name: Creating a kubernetes config map to hold the tpcc benchmark config
+          shell: kubectl create configmap tpcc-config --from-file tpcc.conf -n {{ namespace }}
```

Comment:
Keep config file(tpcc.conf) as variable here.

Reply:
what will be the significance of this change ?